todo-frontend-angular/src/app/todo/todo.service.ts

```diff
@@ -22,5 +24,13 @@ export class TodoService {
     return this.http.get<Todolist>(`${this.url}/${id}`).pipe(map(todolist=>todolist.todos));
 
   }
+  addTodo(newTodo: NewTodoDTO){
+    return this.http.post<Todo>(`${this.todoUrl}`, newTodo,{
+      headers: new HttpHeaders({
```

Comment:
I am pretty sure this cuold be handled on the backend :) Not not a problem right now